contract_execution_examples/factorial.cairo

```diff
@@ -0,0 +1,15 @@
+%lang starknet
+
+// factorial(n) =  n!
+@external
+func factorial(n) -> (result: felt) {
+    if (n == 1) {
+        return (n,);
+    }
+    let (a) = factorial(n - 1);
+    return (n * a,);
+}
+
+func main() {
+    return ();
+}
```

Comment:
I think this isn't used and can be safely removed from the example.

Reply:
for what i have seen, it is used in some tests inside `execute_contract.rs`